Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -185,7 +185,13 @@ var TouchableNativeFeedback = React.createClass({
 
   _handleResponderMove: function(e) {
     this.touchableHandleResponderMove(e);
-    this._dispatchHotspotUpdate(e.nativeEvent.pageX, e.nativeEvent.pageY);
+    if (this.pressInLocation) {
+      // if the pressInLocation is present the elements width and height must be
```

Comment:
> if the pressInLocation is present

This comment just repeats the same information as in `if (this.pressInLocation)`.

When is it present? Is it always present? The old version never checked for it.

Does checking for it mean we only show the hotspot sometimes?